packages/api/src/graphql/article.ts

```diff
@@ -325,12 +325,21 @@ export const GraphQLPublicArticle: GraphQLObjectType<
     comments: {
       type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLPublicComment))),
       resolve: createProxyingResolver(
-        async ({id}, _, {session, authenticateUser, prisma: {comment}}) => {
+        async (
+          {id},
+          _,
+          {session, authenticateUser, prisma: {comment, commentRatingSystemAnswer}}
+        ) => {
           // if session exists, should get user's un-approved comments as well
           // if not we should get approved ones
           const userSession = session?.type === SessionType.User ? authenticateUser() : null
 
-          return getPublicCommentsForItemById(id, userSession?.user?.id ?? null, comment)
+          return getPublicCommentsForItemById(
+            id,
+            userSession?.user?.id ?? null,
+            commentRatingSystemAnswer,
```

Comment:
Should include the "Comment Ratings" or provide a mean value of all ratings. Otherwise, it seems a bit useless for a frontend, no?

Reply:
I don't understand why it would be useless. The mean of all possible answers for me seems to be useless as the frontend will show each answer separately, no?